runelite-client/src/main/java/net/runelite/client/plugins/chatchannel/ChatChannelPlugin.java

```diff
@@ -740,4 +755,27 @@ private void colorIgnoredPlayers(Color ignoreColor)
 			listWidget.setTextColor(ignoreColor.getRGB());
 		}
 	}
+
+	private void showClanOnlineMemberCount(boolean show)
+	{
+		updateClanTitle(WidgetInfo.CLAN_HEADER, client.getClanChannel(), show);
+	}
+
+	private void showGuestClanOnlineMemberCount(boolean show)
+	{
+		updateClanTitle(WidgetInfo.CLAN_GUEST_HEADER, client.getGuestClanChannel(), show);
+	}
+
+	private void updateClanTitle(WidgetInfo widget, ClanChannel channel, boolean show)
+	{
+		Widget header = client.getWidget(widget);
+
+		if (header != null && channel != null)
+		{
+			Widget title = header.getChild(0);
+			int memberCount = channel.getMembers().size();
+			String clanName = channel.getName();
+			title.setText(show ? String.format("%s (%d)", clanName, memberCount) : channel.getName());
```

Comment:
It looks to me like CLAN_CHAT_CHANNEL_REBUILD already assigns this text to the clan name - so all this needs to do is append the count to the existing text.

Reply:
Does that mean that I don't need to worry about resetting the title text if a player toggles off the setting/plugin, since the script does it anyway (and can be updated immediately by clicking the refresh button next to the clan title)? I removed the "show" flag from this, which makes it more simple.

Reply:
We could just client.runScript() one of the listeners maybe to cause it to rebuild. But just ignoring it is probably fine to if there is a button to reload it / it rebuilds so frequently anyway